tests/generative/variable_strategies.py

```diff
@@ -252,10 +254,19 @@ def variable(
 
     cast_to_int = qm_builds(Function.CastToInt, series)
     cast_to_float = qm_builds(Function.CastToFloat, series)
-    
-    date_add_years = qm_builds(Function.DateAddYears, date_series, series)
-    date_add_months = qm_builds(Function.DateAddMonths, date_series, series)
-    date_add_days = qm_builds(Function.DateAddDays, date_series, series)
+
+    # We limit the date operations to series that have not been produced by a CastToInt operation on a
+    # float value.  This avoids errors produced by trying to add/subtract large numbers that were
+    # produced by multiplication (which produces floats) and then casting to int
```

Comment:
This feels to me like the wrong layer to be addressing the issue.  I would imagine we'll find more ways to pass biggish numbers to these date functions, and so we'll have to add further special casing.  And there will probably be other kinds of error that Hypothesis triggers, which we actually consider benign.  So instead of complicating the strategies to avoid triggering those benign errors, I think it might be better to add support for catching and ignoring errors them.

However, this might turn out to be quite a lot of work... so maybe it's OK to have a few special cases!

Reply:
There is one way of handling allowed errors in #852. Is that the kind of thing you had in mind, @inglesp?

Reply:
Yes, exactly.

Reply:
@rebkwok if you like that approach, I'll try to get that PR merged today.

Reply:
That looks good, thanks